### PR TITLE
Anonymous user blog post

### PR DESCRIPTION
**Pull request by mooreds**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205504470611738
  - https://app.asana.com/0/0/1205483163453219

### CODE REVIEW

File: astro/public/img/blogs/anonymous-user/anonymous-user-in-admin-screen.png
[the remark this answers is not in the document]
Comment (alex-fusionauth) [reply astro/public/img/blogs/anonymous-user/anonymous-user-in-admin-screen.png:1]: Screenshot 2023-09-21 at 1 37 48 PM

just added
image

**Review by alex-fusionauth — CHANGES_REQUESTED** [review]
- Image update, maybe
- I couldn't get the actual program to update me from anonymous, but I am probably doing something wrong.
- minor text updates

File: astro/src/content/blog/anonymous-user.mdx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+This reaping process can be run periodically by a scheduled job.
+
+### Privacy Concerns
+
+Make sure you abide by all rules for jurisdictions in which your users live. In particular, make sure you don't collect [any personal data](https://commission.europa.eu/law/law-topic/data-protection/reform/what-personal-data_en) unless you have the processes in place to abide by the GDPR.
+
+Personal data is defined as:
+
+> any information that relates to an identified or identifiable living individual. Different pieces of information, which collected together can lead to the identification of a particular person, also constitute personal data.
```
[the remark this answers is not in the document]
Comment (mooreds) [reply astro/src/content/blog/anonymous-user.mdx:141]: This is quoting from the previously linked doc.

File: astro/public/img/blogs/anonymous-user/anonymous-user-in-admin-screen.png
[the remark this answers is not in the document]
Comment (mooreds) [reply astro/public/img/blogs/anonymous-user/anonymous-user-in-admin-screen.png:1]: Filed an issue about this. https://github.com/FusionAuth/fusionauth-issues/issues/2482

Agree we should fix this, but would rather have the picture be true to what the user will see right now.

**Review by alex-fusionauth — APPROVED** [review]

**Comment by alex-fusionauth** [comment]
LGTM, approved 😄

**Pull request merged**